package.json

```diff
@@ -50,12 +53,15 @@
     "secp256k1": "^3.5.0",
     "sequelize": "^4.37.3",
     "toml": "^2.3.3",
-    "winston": "^3.0.0-rc3"
+    "winston": "^3.0.0-rc3",
+    "yargs": "^11.0.0"
```

Comment:
I was using that `command-line-args` package above for working with command line arguments. Ultimately we should only use one. Thoughts on this?

Reply:
Sorry, forgot about it. 
I guess we can stick with `command-line-args`. I'll remove `yargs` usage.